instant-distance/src/contiguous.rs

```diff
@@ -0,0 +1,240 @@
+use std::marker::PhantomData;
+
+use rand::{rngs::SmallRng, Rng, SeedableRng};
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+use crate::{
```

Comment:
As mentioned in a previous comment, please use one import line per module.

Reply:
Fixed in a later commit